### PR TITLE
feat: Update OVMS server to the latest release image instead of the c…

### PR DESCRIPTION
**Pull request by brian-intel**
…ustom image

## PR Checklist

- [x] Added label to the Pull Request for easier discoverability and search
- [x] Commit Message meets guidelines as indicated in the URL https://github.com/intel-retail/automated-self-checkout/blob/main/CONTRIBUTING.md
- [x] Every commit is a single defect fix and does not mix feature addition or changes
- [x] Unit Tests have been added for new changes
- [x] Updated Documentation as relevant to the changes
- [x] All commented code has been removed
- [ ] If you've added a dependency, you've ensured license is compatible with repository license and clearly outlined the added dependency.
- [ ] PR change contains code related to security
- [ ] PR introduces changes that breaks compatibility with other modules (If YES, please provide details below)

## What are you changing?

Updating the OVMS server image to use the latest release instead of building a custom release. This will significantly reduce setup time for the OVMS server.

## Issue this PR will close

close: #262 

## Test Instructions if applicable

1. clone repository
2. run `make build-ovms-server`
3. latest ovms server image should be pulled into your local system

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -39,15 +39,9 @@ build-ovms-client:
...
-	@echo "Building for OVMS Server HTTPS_PROXY=${HTTPS_PROXY} HTTP_PROXY=${HTTP_PROXY}"
-	$(MAKE) -C model_server docker_build OV_USE_BINARY=0 BASE_OS=ubuntu OV_SOURCE_BRANCH=seg_and_bit_gpu_poc
-	docker build --build-arg HTTPS_PROXY=${HTTPS_PROXY} --build-arg HTTP_PROXY=${HTTP_PROXY} -f $(PWD)/configs/opencv-ovms/models/2022/Dockerfile.updateDevice -t update_config:dev $(PWD)/configs/opencv-ovms/models/2022/.
-
-get-server-code:
-	@if [ -d "./model_server" ]; then echo "clean up the existing model_server directory"; rm -rf ./model_server; fi
-	echo "Getting model_server code"
-	git clone https://github.com/gsilva2016/model_server 
+build-ovms-server:
+	docker pull openvino/model_server:2023.0-gpu
```
Comment (jim-wang-intel) [inline Makefile:43]: will that be a proxy issue if inside intel and we just pull?

Comment (brian-intel) [reply Makefile:43]: it would fail. Let me add the proxy parameter just in case

File: Makefile [hunk 2]
```diff
@@ -40,8 +40,8 @@ build-ovms-client:
 	docker build --build-arg HTTPS_PROXY=${HTTPS_PROXY} --build-arg HTTP_PROXY=${HTTP_PROXY} -t ovms-client:latest -f Dockerfile.ovms-client .
 
 build-ovms-server:
-	docker pull openvino/model_server:2023.0-gpu
-	docker build --build-arg HTTPS_PROXY=${HTTPS_PROXY} --build-arg HTTP_PROXY=${HTTP_PROXY} -f configs/opencv-ovms/models/2022/Dockerfile.updateDevice -t update_config:dev configs/opencv-ovms/models/2022/.
+	HTTPS_PROXY=${HTTPS_PROXY} HTTP_PROXY=${HTTP_PROXY} docker pull openvino/model_server:2023.0-gpu
+	sudo docker build --build-arg HTTPS_PROXY=${HTTPS_PROXY} --build-arg HTTP_PROXY=${HTTP_PROXY} -f configs/opencv-ovms/models/2022/Dockerfile.updateDevice -t update_config:dev configs/opencv-ovms/models/2022/.
```
Comment (jim-wang-intel) [inline Makefile:44]: please avoid sudo for docker here

```suggestion
	docker build --build-arg HTTPS_PROXY=${HTTPS_PROXY} --build-arg HTTP_PROXY=${HTTP_PROXY} -f configs/opencv-ovms/models/2022/Dockerfile.updateDevice -t update_config:dev configs/opencv-ovms/models/2022/.
```

Comment (brian-intel) [reply Makefile:44]: I'll take your change and remove it for now. I noticed there are some permission issues with the model directory so I'll plan to move this dockerfile to another location in a future PR.

**Review by jim-wang-intel — APPROVED** [review]
LGTM

**Review by vli11 — APPROVED** [review]
LGTM

**Review by antoniomtz — APPROVED** [review]

**Pull request merged**